src/components/NavBar.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+import Link from "next/link";
+import Image from "next/image";
+import { usePathname } from "next/navigation";
+
+const LINKS = [
+  { href: "/", label: "Home" },
+  { href: "/joinus", label: "Join Us" },
+  { href: "/board", label: "Board" },
+  { href: "/events", label: "Events" },
+  { href: "/calendar", label: "Calendar" },
+  { href: "/programs", label: "Programs" },
+];
+
+const NavBar = () => {
+  const pathname = usePathname();
+
+  return (
+    <header className="to-ascend-peach bg-gradient-to-r from-white shadow-md">
+      <div className="flex w-full items-center justify-between px-20 py-3">
+        <Link href="/" className="flex items-center gap-3">
+          <Image
+            src="/logo.webp"
+            alt="Logo"
+            width={150}
+            height={100}
+            priority
+          />
+        </Link>
+
+        <nav>
+          <ul className="flex items-center gap-8 text-2xl font-[var(--font-nunito)]">
```

Comment:
```suggestion
          <ul className="flex items-center gap-8 text-2xl">
```

font-nunito is already a defined class
its also the default font so its not needed